### PR TITLE
add badges images for 2 old ProjectOfTheMonths

### LINKED ISSUE #298: Logos cassés
Hey, les logos des toilettes et des arrêts de bus ne s'affichent plus sur la homepage :

image

Help !

### PR DESCRIPTION
**Pull request by nlehuby**
thanks to @overflorian

fix #298

### CODE REVIEW

File: website/images/badges/busstop.svg
Comment (PanierAvide) [inline website/images/badges/busstop.svg:1]: This one appears broken in (at least) Firefox

image

**Comment by overflorian** [comment]
better like that?
badge-arret-bus-mini

**Comment by flacombe** [comment]
Should we merge this @PanierAvide ?

**Comment by PanierAvide** [comment]
Switched the broken image and merged, thank you all :wink:

**Pull request merged**